### PR TITLE
add gradient overlay in background image

### PR DESCRIPTION
**Pull request by Himanshu7692**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,13 +26,14 @@
     <section class="home" id="home">
 
       <!-- home -->
-      <div class="home">
+      <div class="grediant">
```
Comment (PankajVermaGit) [inline index.html:29]: We can not change the predefined classes name.

File: index.html [hunk 2]
```diff
@@ -26,13 +26,14 @@
...
+        
+        <!-- <div style="background-image: url('images/back.jpg'); width="100%" ,height="900px"> -->
```
Comment (PankajVermaGit) [inline index.html:31]: Please remove unwanted comments in html file

File: style.css [hunk 3]
```diff
@@ -100,6 +100,42 @@ span {
...
+  /* background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(0,0,0,0)), color-stop(100%,rgba(0,0,0,0.65))); */
+
+  background-size: cover;
+  color: white;
+  padding: 20px;
+  
+  
+  
+
+/* background-image: linear-gradient(to bottom right ,violet,aqua); */
```
Comment (PankajVermaGit) [inline style.css:129]: please remove comments

File: style.css [hunk 4]
```diff
@@ -100,6 +100,42 @@ span {
...
+  /* background-image: url('images/back.jpg'); */
+  width: 100%;
+  height: 810px;
+  alt:'Computer image';
+  background-image: linear-gradient(to bottom, rgba(245, 246, 252, 0.52), rgba(117, 19, 93, 0.73)),url('images/back.jpg');
+    
+    
+    background-size: cover;
+    color: white;
+    padding: 20px;
```
Comment (PankajVermaGit) [inline style.css:117]: using padding attributes many times

File: style.css [hunk 5]
```diff
@@ -288,6 +324,7 @@ span {
 
 img {
   border-radius: 5px 5px 0 0;
+
```
Comment (PankajVermaGit) [inline style.css:327]: is there any use of it?

**Review by PankajVermaGit — CHANGES_REQUESTED** [review]
Please check the comments

File: index.html (hunk 1 above)
Comment (Himanshu7692) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (Himanshu7692) [reply index.html]: done

File: style.css (hunk 3 above)
Comment (Himanshu7692) [reply style.css]: done

File: style.css (hunk 4 above)
Comment (Himanshu7692) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (Himanshu7692) [reply style.css]: done

**Review by Paarasguptaa — APPROVED** [review]

**Review by PankajVermaGit — APPROVED** [review]
This can be merged now.

**Pull request merged**